Makefile

```diff
@@ -1,15 +1,35 @@
 .PHONY: build clean test all release gen-mocks check-mockery verify-docs clean-docs
 
-OUTPUT = ./riff
-GO_SOURCES = $(shell find . -type f -name '*.go')
-VERSION ?= $(shell cat VERSION)
+ISWIN:=
+ifeq ($(OS),Windows_NT)
+	ISWIN=true
+    OUTPUT = .\riff
+    OUTPUT_EXT = .exe
+	VERSION = $(shell type VERSION)
+	GITDIRTY = $(shell git diff --quiet HEAD || echo dirty)
```

Comment:
I think the quotes in the *nix version are not necessary, so could be factored out